Changes made by media_tree-01-hauppauge.patch are
incompatible with USB tuner MyGica T230C v2. Some mandatory
commands returns -EAGAIN and initialization ends too early.

--- a/drivers/media/tuners/si2157.c
+++ b/drivers/media/tuners/si2157.c
@@ -120,8 +120,8 @@ static int si2157_init(struct dvb_fronte
 	}
	cmd.rlen = 1;
 	ret = si2157_cmd_execute(client, &cmd);
-	if (ret)
+	if (ret && (ret != -EAGAIN))
 		goto err;
 
 	/* Si2141 needs a second command before it answers the revision query */
